edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/routing/CloudEndpoint.cs

```diff
@@ -209,7 +209,7 @@ internal static MetricTags GetTags(string id)
                 return new MetricTags("DeviceId", id);
             }
 
-            public static void MessageCount(string identity) => Util.Metrics.Count(GetTags(identity), EdgeHubToCloudMessageCountOptions);
```

Comment:
So count wouldn't increment? Or it would increment by 1every time?

Reply:
Count would increment by 1 if no value was specified.